app/routes/circulars/circulars.server.ts

```diff
@@ -0,0 +1,248 @@
+/*!
+ * Copyright © 2022 United States Government as represented by the Administrator
+ * of the National Aeronautics and Space Administration. No copyright is claimed
+ * in the United States under Title 17, U.S. Code. All Other Rights Reserved.
+ *
+ * SPDX-License-Identifier: NASA-1.3
+ */
+
+import { tables } from '@architect/functions'
+import { getUser } from '../__auth/user.server'
+
+export type CircularModel = {
+  circularId: number
+  sub: string
+  createdOn: number
+  subject: string
+  body: string
+  submitter?: string
```

Comment:
```suggestion
  submitter: string
```

This should never be null.